scene/main/node.cpp

```diff
@@ -2860,7 +2855,7 @@ void Node::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("get_pause_mode"), &Node::get_pause_mode);
 	ClassDB::bind_method(D_METHOD("can_process"), &Node::can_process);
 	ClassDB::bind_method(D_METHOD("print_stray_nodes"), &Node::_print_stray_nodes);
-	ClassDB::bind_method(D_METHOD("get_position_in_parent"), &Node::get_position_in_parent);
+	ClassDB::bind_method(D_METHOD("get_index"), &Node::get_index);
```

Comment:
This is a pointless diff, no need to remove line 2838 and re-add it here.

Same for the method declarations and definitions.